Common/Securities/Future/FutureCache.cs

```diff
@@ -22,5 +24,21 @@ namespace QuantConnect.Securities.Future
     /// <seealso cref="SecurityCache"/>
     public class FutureCache : SecurityCache
     {
+        /// <summary>
+        /// The current settlement price
+        /// </summary>
+        public decimal SettlementPrice { get; set; }
+
+        /// <summary>
+        /// Will consume the given data point updating the cache state and it's properties
+        /// </summary>
+        /// <param name="data">The data point to process</param>
+        /// <param name="cacheByType">True if this data point should be cached by type</param>
+        protected override void ProcessDataPoint(BaseData data, bool cacheByType)
+        {
+            base.ProcessDataPoint(data, cacheByType);
+
+            SettlementPrice = Price;
```

Comment:
Todo: Switch by data type; look for the settlement tick flag; or if its bar data look for the time closest to the official settlement time.